modules/landing_zone/scripts/helper.js

```diff
@@ -135,20 +136,57 @@ class Helper {
       });
     });
 
-    await Promise.all(
-      Object.keys(jsonComponents).map(key => {
-        processes.push([...terrahubConfig, ...[`terraform.varFile[0]=${jsonComponents[key].toString()}`, '-i', key]]);
-
-        return this.executeWithoutErrors(
-          rootPath, 'terrahub',
-          [...terrahubConfig, ...['terraform', '--delete', '--auto-approve', '--include', key]]
-        );
-      })
-    );
+    await this.updateConfigByComponent(jsonComponents, processes, terrahubConfig, rootPath);
 
     return processes;
   }
 
+  /**
+   * @param {Array} jsonComponents
+   * @param {Array} processes
+   * @param {Array} terrahubConfig
+   * @param {String} rootPath
+   * @return {Promise}
+   */
+  async updateConfigByComponent(jsonComponents, processes, terrahubConfig, rootPath) {
+    await Promise.all(Object.keys(jsonComponents).map(async key => {
+      const re = /\s*\/\*\s*/;
+      const linkList = jsonComponents[key].split(re);
+      if (linkList.length === 1) {
+        processes.push([...terrahubConfig, ...[`terraform.varFile[0]=${jsonComponents[key].toString()}`, '-i', key]]);
+      }
+      else {
+        var res = jsonComponents[key].substring(0, 2);
```

Comment:
```suggestion
        let res = jsonComponents[key].substring(0, 2);
```
As best practice, `let` is better than `var`